pom.xml

```diff
@@ -33,12 +33,12 @@
     -->
     <properties>
         <!-- the following properties set sane defaults when no revision properties are set from the command-line -->
-        <revision>1.15</revision>
+        <revision>1.16</revision>
         <changelist>.0-SNAPSHOT</changelist>
 
         <github.url>scm:git:git@github.com:dockstore/dockstore-support.git</github.url>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <dockstore-core.version>1.15.0-rc.2</dockstore-core.version>
+        <dockstore-core.version>1.16.0-SNAPSHOT</dockstore-core.version>
```

Comment:
Will update to a tag when https://github.com/dockstore/dockstore/pull/5824 is merged